forge-gui/res/cardsfolder/upcoming/tocasia_dig_site_mentor.txt

```diff
@@ -0,0 +1,10 @@
+Name:Tocasia, Digsite Mentor
+ManaCost:1 G W U
+Types:Legendary Creature Human Artificer
+PT:4/3
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddAbility$ Surveil | AddKeyword$ Vigilance | Description$ Creatures you control have vigilance and "{T}: Surveil 1." (To surveil 1, look at the top card of your library You may put that card into your graveyard.)
+SVar:Surveil:AB$ Surveil | Cost$ T | Amount$ 1 | SpellDescription$ Surveil 1." (To surveil 1, look at the top card of your library. You may put that card into your graveyard.)
+A:AB$ ChangeZone | Cost$ 2 G G W W U U ExileFromGrave<1/CARDNAME> | Origin$ Graveyard | Destination$ Battlefield | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Artifact.YouOwn | TgtPrompt$ Return any number of target artifact cards with total mana value 10 or less from your graveyard to the battlefield | MaxTotalTargetCMC$ 10 | GainControl$ True | ActivationZone$ Graveyard | SorcerySpeed$ True | SpellDescription$ Return any number of target artifact cards with total mana value 10 or less from your graveyard to the battlefield. Activate only as a sorcery.
```

Comment:
Maybe: TgtPrompt$ Select any number of target artifact cards with total mana value 10 or less